src/core/store/modules/app/app.actions.ts

```diff
@@ -117,6 +125,47 @@ const getUserAppData = createAsyncThunk<void, { network: Network; route: Route;
   }
 );
 
+/* -------------------------------------------------------------------------- */
+/*                                  Services                                  */
+/* -------------------------------------------------------------------------- */
+
+const checkServicesStatus = createAsyncThunk<void, void, ThunkAPI>(
+  'app/checkServicesStatus',
+  async (_arg, { dispatch, extra }) => {
+    const { YEARN_ALERTS_API } = extra.config;
+    try {
+      const servicesStatusResponse = await get(`${YEARN_ALERTS_API}/health`);
```

Comment:
nit: A bit cleaner IMO to destructure here
```suggestion
      const { status, data } = await get(`${YEARN_ALERTS_API}/health`);
```

Reply:
refactored here https://github.com/yearn/yearn-finance-v3/pull/674/commits/06ec326e7ff0dd9c530a2f337abbf14fc3ccf217